### DISCUSSION TITLE
How do I send the code version?

### DISCUSSION THREAD

**Discussion by robbporto** [rollbar/rollbar-react#9] [Q&A]
Is there an option to send the code version?

**Comment by eudaimos** [comment]
Yes @robbporto.

All options that you provide to the `Provider`'s `config` prop are passed directly through to the Rollbar instance that the Provider constructs for you.  In the README instructions you see the `const rollbarConfig` declaration, you can add the code version there. Here are the [Rollbar JS configuration docs](https://docs.rollbar.com/docs/rollbarjs-configuration-reference) for your convenience.

The other way to do it is to construct your own Rollbar instance and pass that to the `Provider` using the `instance` prop, also further down in the README.

I hope that helps 😄
